server/mvc/models/User.js

```diff
@@ -8,29 +8,21 @@ var config,
 
 module.exports = function(){
   var self = this,
-    document = new ThinkyModel({}),
-    exists = false;
+    document = new ThinkyModel({});
 
   this.findByUserId = function(userId){
```

Comment:
I agree on this detail :)
As we're already calling `User`, `getById` would be the most sensible of the two.